workers/unity/Packages/com.improbable.gdk.buildsystem/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+using System.Runtime.CompilerServices;
```

Comment:
do we need this file?

Reply:
Yes, so the generated build system code can see the `internal` classes (See below in this file)